cypress/utils/ctsFields.js

```diff
@@ -19,7 +19,7 @@ export const fieldMap = {
     "TrialID": "trialId",
     "TrialInvestigator": "inv",
     "LeadOrganization": "component-unique-id-20",
-    "Keywords": "keywordPhrases",
-    "TrialPhase": "trialphase"
+    "TrialPhase": "trialphase",
+    "Keywords": "keyword"
```

Comment:
Please revert ctsFields to the original -we do not want to touch it to avoid merge conflicts.
If your locators do not work with values given in ctsField, then just hardcoded locators in the specific index.js